content/en/case-studies/gw-discov.md

```diff
@@ -0,0 +1,62 @@
+
+
+### Key Objectives
+
+* Though its [mission](https://www.ligo.caltech.edu/page/what-is-ligo) is to detect gravitational waves from some of the most violent and energetic processes in the Universe, the data LIGO collects may have far-reaching effects on many areas of physics including gravitation, relativity, astrophysics, cosmology, particle physics, and nuclear physics.
+* Crunch observed data via numerical relativity computations that involves complex maths in order to discern signal from noise, filter out relevant signal and statistically estimate significance of observed data 
+* Data visualization so that the binary / numerical results can be comprehended.
+ 
+
+### The Challenges 
+
+* **Computation**
+
+    Gravitational Waves are hard to detect as they produce a very small effect and have tiny interaction with matter.  Processing and analyzing all of LIGO's data requires a vast computing infrastructure.After taking care of noise, which is billions of times of the signal, there is still very complex relativity equations and huge amounts of data which present a computational challenge: [O(10^7) CPU hrs needed for binary merger analyses](https://youtu.be/7mcHknWWzNI) spread on 6 dedicated LIGO clusters 
+
+* **Data Deluge**
+
+    As observational devices become more sensitive and reliable, the challenges posed by data deluge and finding a needle in a haystack rise multi-fold. LIGO generates terabytes of data every day! Making sense of this data requires an enormous effort for each and every detection. For example, the signals being collected by LIGO must be matched by supercomputers against hundreds of thousands of templates of possible gravitational-wave signatures.
+
+* **Visualization**
+
+    Once the obstacles related to understanding Einstein’s equations well enough to solve them using supercomputers are taken care of, the next big challenge was making data comprehensible to the human brain. Simulation modeling as well as  signal detection requires effective visualization techniques.  Visualization also plays a role in lending more credibility to numerical relativity in the eyes of pure science aficionados.
```

Comment:
The last sentence is a little confusing to me. I'm also left to wonder whether "visualization" is really one of the main challenges. The image below this paragraph is straightforward to make with Matplotlib. Should this paragraph be rephrased or removed?

Reply:
ok- will give it another try.  What I wanted to say is - agility in visualisation - this capability is achieved via NumPy through fast array processing when newer data/observations/insights/analysis need to be reimaged.